WD-Firmware/examples/build/SConscript_psp

```diff
@@ -15,34 +15,38 @@
  limitations under the License.
 '''
 import os
+import utils
 Import('Env')
 
 out_dir = os.path.join(Env['OUT_DIR_PATH'], 'psp')
-if not os.path.exists(out_dir):
-    os.makedirs(out_dir)
+utils.createFolder(out_dir)
 
 list_files_c=[

```

Comment:
add to the comment header short syntax description

Reply:
done